test/functionalTest/cases/0000_subs_attrs_deletion/delete_attribute_not_in_condition.test

```diff
@@ -166,15 +166,16 @@ Date: REGEX(.*)
             "http": {
                 "url": "http://localhost:REGEX(\d+)/notify"
             },
-            "onlyChangedAttrs": false
+            "lastNotification": "REGEX(.*)",
+            "lastSuccess": "REGEX(.*)",
+            "lastSuccessCode": 200,
+            "onlyChangedAttrs": false,
+            "timesSent": 1
         },
         "status": "active",
         "subject": {
             "condition": {
-                "attrs": [
-                    "B",
-                    "C"
-                ],
+                "attrs": [],
```

Comment:
Revert this change. Note tests are failing due to this, https://github.com/telefonicaid/fiware-orion/actions/runs/4919583948/jobs/8876244422

```
----------- Failing tests ------------------
  o  0000_subs_attrs_deletion/delete_attribute_not_in_condition.test
  o  0501_cors/allowed_methods.test
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4321/commits/c7478f401c0466246950316809c9ec5521bb1ee1